src/components/EditorsColumn.jsx

```diff
@@ -43,18 +43,22 @@ export default class EditorsColumn extends React.Component {
   }
 
   _storeEditorRef(index, editor) {
-    this.editorRefs.set(index, editor);
+    this.editorRefs[index] = editor;
   }
 
   _storeDividerRef(index, divider) {
-    this.dividerRefs.set(index, divider);
+    this.dividerRefs[index] = divider;
   }
 
   _handleEditorDividerDrag(index, _, {deltaY, lastY, y}) {
-    const {onUpdateFlex} = this.props;
-    const editorHeights = getNodeHeights(this.editorRefs);
-    const dividerHeights = getNodeHeights(this.dividerRefs);
-    onUpdateFlex({deltaY, dividerHeights, editorHeights, index, lastY, y});
+    this.props.onUpdateFlex({
```

Comment:
How about pushing the `dividerDrag` abstraction down and make the prop name `on-divider-drag`?